.travis.yml

```diff
@@ -155,7 +155,7 @@ before_script:
   # Bootstrap everything (including letor, which isn't done
   # by default), then configure using our chosen compiler.
   - travis_retry ./bootstrap xapian-core xapian-applications/omega swig xapian-letor xapian-bindings
-  - ./configure $confargs CC="$USE_CC" CXX="$USE_CXX"
+  - ./configure --enable-letor $confargs CC="$USE_CC" CXX="$USE_CXX"
```

Comment:
I don't think this should be handled in a CI-specific way.

If someone is doing development work on letor then we really want them to be working in a source tree with the bindings to letor also enabled - otherwise they might commit (or submit a PR for) changes which break the letor bindings without knowing it, because `make check` worked fine in their local tree because it wasn't testing the letor bindings.

So I think we want to handle this in `bootstrap` instead - e.g. if `xapian-letor` is being bootstrapped, then add `--enable-letor` to the configure options.

That way it gets enabled for the CI automatically (because that bootstraps `xapian-letor`) but also gets automatically enabled for developers working on `xapian-letor`.

Reply:
Do we have a way in `bootstrap` of adding `--enable-letor` only to configure for the bindings? That would be preferable. Looks like it should be possible around line 900 of bootstrap in the `END_OF_CONFIGURE` heredoc.

Reply:
Line 900 is for SWIG - line 918 is the case for `xapian-bindings`.

It'd be a bit cleaner to target just `xapian-bindings` but it doesn't really matter if it's specified more broadly though - we already deliberately run `configure` with `--disable-option-checking` which turns off checks for unknown `--enable-*`, etc (so that you can use things like `./configure --enable-log` with the top level configure without getting warnings.)

Reply:
Done.

Reply:
> Line 900 is for SWIG - line 918 is the case for `xapian-bindings`.

Gosh, my `HEAD` seems to be *old*…